src/nidmm/system_tests/test_system_nidmm.py

```diff
@@ -244,3 +244,64 @@ def test_configure_waveform_acquisition(session):
     assert session.range == 100
     assert session.waveform_rate == 100000
     assert session.waveform_points == 400
+
+
+def test_fetch_waveform(session):
+    session.configure_waveform_acquisition(nidmm.Function.WAVEFORM_VOLTAGE, 10, 1000, 10)
+    with session.initiate():
+        number_of_points_to_read = 10
+        measurements,actual_number_of_points = session.fetch_waveform(number_of_points_to_read)
+        assert len(measurements) == number_of_points_to_read
```

Comment:
Can we instead do:
'assert isinstance(measurements[1], float)'

Reply:
I think that this should be 'in addition to' rather than 'instead of'. Verifying that the length of the array matches what was  returned for number of points read is a valid test.